### PR TITLE
Cache results until the next day

### LINKED ISSUE #1: Add caching for detailed stats
Since the results are only updated once a day and because the API has a liberal definition of how and what happens when the limit is being reached, it's probably safer to cache the results for a day.

From the documentation:

> Throttling Behaviour
Requests are tracked by the minute. It is considered best practice to not exceed 30 requests per minute. Violation of this limit could result in degraded performance up to and including the revocation of the API key.

### PR DESCRIPTION
**Pull request by mgibeau**
closes #1

### CODE REVIEW

File: src/details/actions.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
 import moment from 'moment';
 import numeral from 'numeral';
+import hash from 'string-hash';
+import ls from 'local-storage';
 
 import * as d from './actionTypes';
 
 // Mock data to avoid hammering the API
 // import json from './sample/usageRecords.json';
```
Comment (Greg-Hamel) [inline src/details/actions.js]: Should this be removed as it is commented out?

File: src/details/actions.js [hunk 2]
```diff
@@ -63,7 +73,7 @@ export const getUsageDetails = (periodEnd, skip) => {
   return dispatch => {
     dispatch({ type: d.LOADING })
 
-    const past30Days = moment(periodEnd).subtract(1, 'months').format()
+    const past30Days = moment(periodEnd).startOf('day').subtract(1, 'months').format()
 
     // Return the mock data
     // dispatch(computeTrends(json))
```
Comment (Greg-Hamel) [inline src/details/actions.js]: Should this be removed as it is commented out?

Comment (Greg-Hamel) [reply src/details/actions.js]: it is also a duplicate from line `93` and `102` which seem to be run no matter what.

Recommend deletion.

**Review by Greg-Hamel — CHANGES_REQUESTED** [review]
Minor changes related to commented-out code.
Recommend approval once changed.

File: src/details/actions.js (hunk 1 above)
Comment (mgibeau) [reply src/details/actions.js]: Sure, l guess this should be transferred to unit test, I've created #10 to take care of it.

File: src/details/actions.js (hunk 2 above)
Comment (mgibeau) [reply src/details/actions.js]: Yup, this ties in to the import above, will get rid of it for now

**Review by Greg-Hamel — APPROVED** [review]

**Pull request merged**